libs/auth-api-lib/migrations/1672932758825-add-api-scope-user-name-field.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((transaction) =>
+      Promise.all([
+        queryInterface.addColumn(
+          'api_scope_user',
+          'name',
+          {
+            type: Sequelize.STRING,
+            allowNull: true,
+          },
+          { transaction },
+        ),
+      ]),
+    )
```

Comment:
You can simplify your migration a bit, and also just join those two into a single file. And modify the field to not allow null when you have set default values.
```suggestion
    return queryInterface.sequelize.transaction(async (transaction) => {
        await queryInterface.addColumn(
          'api_scope_user',
          'name',
          {
            type: Sequelize.STRING,
            allowNull: true,
          },
          { transaction },
        )
        
        await queryInterface.sequelize.query(
            `
            UPDATE api_scope_user
            SET name = 'NAME_NOT_SET'
            WHERE name IS NULL`,
            { transaction },
        )
        
        await queryInterface.changeColumn(
          'api_scope_user',
          'name',
          {
              type: Sequelize.STRING,
              allowNull: false,
          },
          { transaction },
        )
        
    })
```